Test failures seem to be related to light component


```
 =========================== short test summary info ============================
FAILED tests/components/light/test_device_trigger.py::test_if_fires_on_state_change_with_for[pyloop]
Error: Process completed with exit code 1.
```

--- a/homeassistant/components/atag/config_flow.py
+++ b/homeassistant/components/atag/config_flow.py
@@ -1,4 +1,4 @@
-"""Config flow for the Atag component."""
+"""Config flow for the Atag component"""
 import pyatag
 import voluptuous as vol
 

--- a/homeassistant/components/atag/config_flow.py
+++ b/homeassistant/components/atag/config_flow.py
@@ -1,4 +1,4 @@
-"""Config flow for the Atag component"""
+"""Config flow for the Atag component."""
 import pyatag
 import voluptuous as vol
 

